src/primitives/transaction.h

```diff
@@ -293,7 +293,7 @@ inline void SerializeTransaction(TxType& tx, Stream& s, Operation ser_action, in
     const bool fAllowWitness = !(nVersion & SERIALIZE_TRANSACTION_NO_WITNESS);
 
     READWRITE(*const_cast<int32_t*>(&tx.nVersion));
-    unsigned char flags = 0;
+    unsigned char _flags = 0;
```

Comment:
sflags  (this was less obvious to me, but my suggestion and @sipa thought it was okay)